assets/assets.go

```diff
@@ -0,0 +1,13 @@
+package assets
+
+import (
+	"embed"
+)
+
+//go:embed *
```

Comment:
```suggestion
//go:embed *.yaml rbac/*.yaml
```

I recommend to explicitly define what files you are going to include in the embed.FS to prevent adding `assets.go` there.

Reply:
Fixed